src/overlays/actors/ovl_En_Wf/z_en_wf.h

```diff
@@ -6,9 +6,88 @@
 
 struct EnWf;
 
+typedef void (*EnWfActionFunc)(struct EnWf*, GlobalContext*);
+
+typedef enum {
+    /*  0 */ WOLFOS_LIMB_NONE,
+    /*  1 */ WOLFOS_LIMB_ROOT,
+    /*  2 */ WOLFOS_LIMB_BACK_LEFT_THIGH,
+    /*  3 */ WOLFOS_LIMB_BACK_LEFT_SHIN,
+    /*  4 */ WOLFOS_LIMB_BACK_LEFT_PASTERN,
+    /*  5 */ WOLFOS_LIMB_BACK_LEFT_PAW,
+    /*  6 */ WOLFOS_LIMB_TAIL,
+    /*  7 */ WOLFOS_LIMB_ABDOMEN,
+    /*  8 */ WOLFOS_LIMB_BACK_RIGHT_THIGH,
+    /*  9 */ WOLFOS_LIMB_BACK_RIGHT_SHIN,
+    /* 10 */ WOLFOS_LIMB_BACK_RIGHT_PASTERN,
+    /* 11 */ WOLFOS_LIMB_BACK_RIGHT_PAW,
+    /* 12 */ WOLFOS_LIMB_THORAX,
+    /* 13 */ WOLFOS_LIMB_FRONT_RIGHT_UPPER_LEG,
+    /* 14 */ WOLFOS_LIMB_FRONT_RIGHT_LOWER_LEG,
+    /* 15 */ WOLFOS_LIMB_FRONT_RIGHT_CLAW,
+    /* 16 */ WOLFOS_LIMB_HEAD_ROOT,
+    /* 17 */ WOLFOS_LIMB_HEAD,
+    /* 18 */ WOLFOS_LIMB_EYES,
+    /* 19 */ WOLFOS_LIMB_FRONT_LEFT_UPPER_LEG,
+    /* 20 */ WOLFOS_LIMB_FRONT_LEFT_LOWER_LEG,
+    /* 21 */ WOLFOS_LIMB_FRONT_LEFT_CLAW,
+    /* 22 */ WOLFOS_LIMB_MAX
+} EnWfLimb;
+
+typedef enum {
+    /*  0 */ WOLFOS_ACTION_WAIT_TO_APPEAR,
+
+    /*  2 */ WOLFOS_ACTION_DIE = 2,
```

Comment:
i would remove the breaks for the missing entries. assigning the values manually is what will point the fact out that some are missing